modules/ecs-service/ecs-service.tf

```diff
@@ -39,7 +39,8 @@ locals {
       dependsOn           = []
       mountpoints         = var.mountpoints
       secrets             = var.secrets
-      environments        = var.environments
+      environments        = []
```

Comment:
why is environments = var.environments replaced with [] ? I think we still want to keep the normal functionality next to the new functionality?

Reply:
Yes, that's a leftover from some testing. It shouldn't be there